Azure/azure-cli
# [Storage] Fix #16979: `az storage container create` fails when providing storage container metadata

**Description**

Fix #16979 

`az storage container create` has command validator `validate_client_auth_parameter` which will suppress all argument validators including `validate_metadata`. As a result, `metadata` can't be parsed properly.

**Testing Guide**

`az storage container create -n testcontainer --metadata k1=v1 k2=v2`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
